src/renderers/shaders/ShaderChunk/cube_uv_reflection_fragment.glsl.js

```diff
@@ -104,7 +104,15 @@ export default /* glsl */`
 		uv.x *= CUBEUV_TEXEL_WIDTH;
 		uv.y *= CUBEUV_TEXEL_HEIGHT;
 
-		return texture2D( envMap, uv ).rgb;
+		#ifdef TEXTURE_LOD_EXT
```

Comment:
Would doing this instead work?

```glsl
#ifdef texture2DGradEXT
```

Reply:
I think it's equivalent, yes, but this was the how it was done elsewhere in the code, specifically here: https://github.com/mrdoob/three.js/blob/dev/src/renderers/shaders/ShaderChunk/transmission_pars_fragment.glsl.js#L60

Reply:
I see...